IBAnimatableApp/TimeLineTableViewController.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  TimeLineTableViewController.swift
+//  IBAnimatable
+//
+//  Created by Eric Marchand on 31/03/2017.
+//  Copyright © 2017 IBAnimatable. All rights reserved.
+//
+
+import UIKit
+import IBAnimatable
+
+public class TimeLineTableViewController: AnimatableTableViewController {
+
+  override public func viewDidLoad() {
+    super.viewDidLoad()
+
+    // Install action on refresh
+    self.refreshControl?.addTarget(self, action: #selector(TimeLineTableViewController.refresh(_:)), for: .valueChanged)
+  }
+
+  public func refresh(_ refreshControl: UIRefreshControl) {
+    // could update attributedTitle of refreshControl here
+
+    // Simulate an asynchrone refresh, could be a network request...
+    let time: TimeInterval = 5
+    self.updateMessage(refreshControl: refreshControl, time: time)
+    DispatchQueue.background.after(time) {
```

Comment:
To simulate a good practice, doing a  job in background like get data from network